### PR TITLE
22 order factory

### PR DESCRIPTION
**Pull request by Anulfo**
## Status
**READY**

## Testing

[ x] I have added passing tests for all new functionality.

[ ] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[ x] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Create a Order Factory to retrieve all the orders corresponding to the selected customer

## Resolves Issue Number
#22
## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any

```sh
git pull --prune
git checkout 22-OrderFactory
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Got to Visual Studio.
2. Open the solution.
3. Run the tests

## Impacted Files in Application
List general components of the application that this PR will affect:

* src/customer-service/Factories/OrderFactory.cs
*src/customer-service/customer-service-test/OrderFactoryTests.cs

### CODE REVIEW

File: src/customer-service/Factories/OrderFactory.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+{
+
+    /**
++     * Class: OrderFactory
++     * Purpose: Creates a connection between the database and the user to retrieve every order for the required customer
++     * Author: Anulfo Ordaz
++     * Methods:
++     *     
++     *     public static OrderFactory Instance - creates an instance of the order factory
++     *     public Customer ActiveCustomer - This method sets an active customer for the program
```
Comment (MattKraatz) [inline src/customer-service/Factories/OrderFactory.cs]: This method isn't on this factory :-)

**Review by MattKraatz — APPROVED** [review]
LGTM, one quick change on your documentation

**Comment by matthamil** [comment]
LGTM :boat:

File: customer-service-tests/OrderFactoryTests.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        {
+            var instance = OrderFactory.Instance;
+            Assert.NotNull(instance);
+
+        }
+
+        [Fact]
+
+        public void OrderFactoryIsSingleton()
+        {
```
Comment (garrett-vangilder) [inline customer-service-tests/OrderFactoryTests.cs:27]: dont need?

File: src/customer-service/.vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (garrett-vangilder) [inline src/customer-service/.vscode/launch.json:1]: should not be pushed up

**Review by garrett-vangilder — APPROVED** [review]
LGTTeamCharles 🤘

**Pull request merged**